### PR TITLE
[ossf-scorecard] Add first fuzzer

### PR DESCRIPTION
**Pull request by AdamKorcz**
Adds the first fuzzer for ossf-scorecard to get the fuzzing started.

@justaugustus @naveensrinivasan

### CODE REVIEW

**Review by DavidKorczynski — APPROVED** [review]

File: projects/ossf-scorecard/yaml_fuzzer.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package policy
+
+func FuzzParseFromYAML(data []byte) int {
```
Comment (naveensrinivasan) [inline projects/ossf-scorecard/yaml_fuzzer.go:18]: Thanks, I would rather have the fuzzing code within the scorecard. Is there a specific reason to keep idt outside?

Comment (naveensrinivasan) [reply projects/ossf-scorecard/yaml_fuzzer.go:18]: If we change the API now someone in oss-fuzz would have to maintain this, with go 1.18 support fuzzing would be have been much easier compared to go-fuzz

Comment (AdamKorcz) [reply projects/ossf-scorecard/yaml_fuzzer.go:18]: No problem. I will make a PR upstream with a Go 1.18 fuzzer shorty.

Comment (AdamKorcz) [reply projects/ossf-scorecard/yaml_fuzzer.go:18]: Go 1.18 fuzzer added here: https://github.com/ossf/scorecard/pull/1631

**Pull request merged**